ContextApp/src/main/java/com/github/migi_1/ContextApp/EventButtonPressActivity.java

```diff
@@ -0,0 +1,27 @@
+package com.github.migi_1.ContextApp;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+
+/**
+ *
+ * @author Nils
+ */
+public class EventButtonPressActivity extends Activity {
+        @Override
+        public void onCreate(Bundle savedInstanceState) {
+            super.onCreate(savedInstanceState);
+            setContentView(R.layout.android_event_button);
+            Button returnButton = (Button) findViewById(R.id.event1_button);
+            returnButton.setOnClickListener(new View.OnClickListener() {
+
+                @Override
+                public void onClick(View v) {
+                    finish();
+                }
+            });
+        }
+}
```

Comment:
This file can be removed from this PR, it was added in a merge and doesn't do anything in this PR.